src/ext/framework/mod.rs

```diff
@@ -477,10 +477,13 @@ impl Framework {
                         let after = self.after.clone();
                         let groups = self.groups.clone();
 
+                        let content = message.content[position..].to_owned();
+                        let content = content.trim();
+
                         let args = if command.use_quotes {
```

Comment:
If you move the `content` binding to within a new scope within `args`, then you can avoid `to_owned()`.

e.g.:
```rust
let args = {
    let content = ....

    if command.use_quotes {
        //
    } else {
        //
    }
};
```